packages/tools/lib/postcss-css-to-esm/index.js

```diff
@@ -60,7 +60,7 @@ module.exports = function (opts) {
 
 	return {
 		postcssPlugin: 'postcss-css-to-esm',
-		Once (root) {
+		OnceExit(root) {
```

Comment:
Are this and the other similar change intentional?

Reply:
the `Once` hook runs too early and the variables are not renamed. They are renamed in the generated CSS, but this hook runs too early and does not see the result. As a side effect, duplicate parameters are now correctly removed reducing the output of the parameters-bundle from ~1900 lines to ~1400 lines